src/cli/backup/exchange.go

```diff
@@ -148,7 +148,7 @@ func createExchangeCmd(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	if err := validateExchangeBackupCreateFlags(utils.User, utils.CategoryData); err != nil {
+	if err := validateExchangeBackupCreateFlags(utils.UserFV, utils.CategoryDataFV); err != nil {
```

Comment:
noob q: What's FV?

Reply:
Flag Value.  Not the best namespacing, but I've bumped into the "is this the foo populated by the flag, or the foo that I passed in as a param" bug a couple times now.  So having _something_ as a naming convention to separate values could identify those mixups.